addons/binding/org.openhab.binding.co2signal/src/main/java/org/openhab/binding/co2signal/internal/json/CO2SignalJsonData.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.co2signal.internal.json;
+
+/**
+ * The {@link CO2SignalJsonData} is responsible for storing
+ * the "data" node from the co2signal.com JSON response
+ *
+ * @author Jens Viebig - Initial contribution
+ */
+public class CO2SignalJsonData {
+
+    private double carbonIntensity;
+    private double fossilFuelPercentage;
+
+    public CO2SignalJsonData() {
```

Comment:
Default constructor is not needed

Reply:
Disagree. Gson encourages to add a no-args constructor. It seems to be useful for reflection